libs/domains/environments/feature/src/lib/environment-state-chip/environment-state-chip.spec.tsx

```diff
@@ -0,0 +1,41 @@
+import { renderWithProviders } from '@qovery/shared/util-tests'
+import EnvironmentStateChip from './environment-state-chip'
+
+jest.mock('../hooks/use-deployment-status/use-deployment-status', () => {
+  return {
+    ...jest.requireActual('../hooks/use-deployment-status/use-deployment-status'),
+    useDeploymentStatus: () => ({
+      data: {
+        state: 'BUILDING',
```

Comment:
Why you didn't use enum in your tests? boundaries issue?

Reply:
Lazyness :sweat_smile:

Reply:
https://github.com/Qovery/console/pull/757/commits/92853280d057f7ea97a18cc7f29fe33d50826d93

Reply:
Enums in jest mock is a mess
https://stackoverflow.com/questions/74657645/how-do-i-use-typescript-and-jest-requireactual-with-named-exports
https://github.com/nrwl/nx/issues/15514